components/edit_category_modal/edit_category_modal.tsx

```diff
@@ -4,14 +4,17 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import {GenericModal} from '@mattermost/components';
```

Comment:
Is this change needed or can we still use the old one? It seems like the one from `components/generic_modal` includes some CSS that we might otherwise miss when importing from the components package right now

Reply:
Yeah reverted that

Reply:
reverted to old one